### PR TITLE
New PR for moolight game streaming

### PR DESCRIPTION
**Pull request by lclol**
New proposal for Moonligt v 8.11

My excuses, once again, but I think I'm making progress... 

This time, the changes must be clear and well made :)

### CODE REVIEW

File: classic/com.limelight.json [hunk 1]
```diff
@@ -14,6 +14,16 @@
...
+            "name": "8.11",
+            "versionCode": 216,
+            "uuid": "eddb769c-1e16-4a22-91e3-5a3c60c503fd",
+            "date": "2020-03-25T00:00:00Z",
+            "url": "https://github.com/moonlight-stream/moonlight-android/releases/download/v8.11/app-nonRoot-release.apk",
+            "size": 5397046,
+            "md5sum": "d23cf9469f4055e511474f9554a2e4d6",
+            "latest": true
+        },
+		{
```
Comment (cweiske) [inline classic/com.limelight.json:26]: indention here is wrong.
You also need to set "latest" to false in the older release.

**Comment by cweiske** [comment]
Btw, it is not necessary to close an PR just to make changes. Just do your changes, commit+push them.

**Comment by lclol** [comment]
Corrected :)

File: classic/com.limelight.json [hunk 2]
```diff
@@ -14,6 +14,16 @@
...
+            "name": "8.11",
+            "versionCode": 216,
+            "uuid": "eddb769c-1e16-4a22-91e3-5a3c60c503fd",
+            "date": "2020-03-25T00:00:00Z",
+            "url": "https://github.com/moonlight-stream/moonlight-android/releases/download/v8.11/app-nonRoot-release.apk",
+            "size": 5397046,
+            "md5sum": "d23cf9469f4055e511474f9554a2e4d6",
+            "latest": true
+        },
+	{
```
Comment (cweiske) [inline classic/com.limelight.json:26]: wrong indentation. I'll fix this myself

**Comment by cweiske** [comment]
Thank you for the patch - merged.

My server has the new version now.

**Comment by cweiske** [comment]
@lclol Please test that on your OUYA. The OUYA has only partial HTTPS support, so it might not work.

**Pull request closed without merging**